src/ripple/app/main/Main.cpp

```diff
@@ -488,6 +495,81 @@ int run (int argc, char** argv)
             crypto_prng().load_state(entropy.string ());
     }
 
+    if (vm.count("vacuum"))
+    {
+        DatabaseCon::Setup dbSetup = setup_DatabaseCon(*config);
```

Comment:
not critical, but this `vacuum` code seems like a good candidate for a small separate function, possibly placed in `SociDB.cpp` (where there are already a few util-like functions). Up to you, but it might also make it possible to write a test for it then as well.